examples/dbus-info/atomicClient.html

```diff
@@ -0,0 +1,220 @@
+              }
+
+              else if(command.indexOf("--images") == 1) {
+                call = proxy.Scan(scan_targets, scanner, scan_type, rootfs, false, true, false);
+              }
+
+              else if(command.indexOf("--containers") == 1) {
+                call = proxy.Scan(scan_targets, scanner, scan_type, rootfs, false, false, true);
+              }
+
+              else {
+                call = proxy.Scan([command[1]], scanner, scan_type, rootfs, false, false, false);
+              }
+
+              call.done(function(result) {
+                console.log(JSON.stringify(result));
+              });
+
+              call.fail(function(error) {
+                console.warn(error);
+              });
+            }
+
+            function run_vulnerable_info() {
+              var call = proxy.VulnerableInfo();
+              call.done(function(result) {
+                console.log(result);
+              });
+              call.fail(function(error) {
+                console.warn(error);
+              });
+            }
+
+            function run_update(command) {
+              var call = proxy.Update(command[1]);
+              call.done(function() {
+                console.log("Success");
+              });
+
+              call.fail(function(error) {
+                console.warn(error);
+              });
+            }
+
+            function run_images() {
+              var call = proxy.Images();
+              var text = "Repository                                                                                                                                       Last Scanned\n";
+              call.done(function(result) {
+                response = JSON.parse(result);
+                for (var i = 0; i < response.length; i++) {
+                  text += response[i]["repo"] + "            ";
+                  if ("Time" in response[i]["vuln_info"]) {
+                    text += response[i]["vuln_info"]["Time"] + " ";
+                  }
+
+                  if(response[i]["vulnerable"]) {
+                    text += "*\n";
+                  }
+                  else {
+                    text += "\n";
+                  }
+                }
+                alert(text);
+              });
+
+              call.fail(function(error) {
+                console.warn(error);
+              });
+            }
+
+            //Depending on what is entered, it will send the command to the appropriate function to be parsed.
+            function run_command() {
+              var command = (input.val()).split(" ");
+              var call;
+              if(command[0].toLowerCase() == "scan") {
+                if(command.indexOf("--list") != -1) {
+                  run_scan_list();
+                }
+                else {
+                  run_scan(command);
+                }
+              }
+
+              else if(command[0].toLowerCase() == "update") {
```

Comment:
sure yeah i was trying to make sure everything was working initially for the POC. Now that the step is over, I can look into the buttons and options.